corehq/sql_db/util.py

```diff
@@ -290,7 +324,5 @@ def select_db_for_read(weighted_dbs):
             weights.append(weight)
 
     if dbs:
-        # normalize weights of remaining dbs
-        total_weight = sum(weights)
-        normalized_weights = [float(weight) / total_weight for weight in weights]
-        return random.choice(dbs, p=normalized_weights)
+        choices_func = random.choices if sys.version_info >= (3, 6) else choices_backport
```

Comment:
MIght be nicer to have something like:

```python
try:
   from random import choices
except ImportError:
   def choices():
        ....
```

could also be extracted into another file.

Reply:
+1

Reply:
Excellent idea.